examples/pipeline-preference-dataset-llmpool.py

```diff
@@ -108,7 +108,8 @@ def load_gpt_4(task: UltraFeedbackTask) -> LLM:
             ]
         ),
         labeller=ProcessLLM(
-            task=UltraFeedbackTask.for_instruction_following(), load_llm_fn=load_gpt_4
+            task=UltraFeedbackTask.for_instruction_following(),
+            load_llm_fn=load_gpt_4,  # type: ignore
```

Comment:
instead of adding this `# type: ignore`, could you change the `task` argument type of `load_gpt_4` function to `Task`?